site/hxapi/asynchronous-bookings.md

```diff
@@ -0,0 +1,169 @@
+---
+
+---
+
+# Asynchronous Bookings
+
+## Background
+
+We now offer products that use an asynchronous booking process. This approach enables partners to handle a broader range of products, including those requiring longer confirmation times or complex supplier interactions. By adopting asynchronous bookings, partners can unlock new opportunities for growth, access a more extensive inventory, and offer customers a seamless booking experience.
+<br>
+<br>
+Asynchronous bookings require a different approach compared to synchronous bookings to ensure smooth integration and maintain customer satisfaction.
+<br>
+<br>
+
+## Enabling Async Product Availability  
+
+To view all supply, including async products, partners can now specify whether they want these products included in the availability response.
+
+### Availability Request  
+
+Partners must include a new parameter in the availability request to indicate whether to include async products.  
+
+| Name                    | Data Type | Format  | Mandatory? | Additional Information                                    |
+|-------------------------|-----------|---------|------------|----------------------------------------------------------|
+| includeAsyncProducts    | Boolean   | `true`/`false` | N | If `true`, availability includes async products. Default is `false`. |
+
+### UK Products Availability Request
+
+{% codetabs %}
+{% codetab XML %}
+```
+https://api.holidayextras.co.uk/v1/carpark/LGW?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2020-01-07&ArrivalTime=1200&DepartDate=2020-01-08&DepartTime=1200&NumberOfPax=1&includeAsyncProducts=true
+```
+{% endcodetab %}
+{% codetab JSON %}
+```https://api.holidayextras.co.uk/v1/carpark/LGW.js?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2020-01-07&ArrivalTime=1200&DepartDate=2020-01-08&DepartTime=1200&NumberOfPax=1&includeAsyncProducts=true
+```
+Note the `.js` extension.
+{% endcodetab %}
+{% endcodetabs %}
+
+### European Products Availability Request
+
+{% codetabs %}
+{% codetab XML %}
+```
+https://api.holidayextras.co.uk/v1/carpark/MUC?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2017-12-01&ArrivalTime=1200&DepartDate=2017-12-08&DepartTime=1200&NumberOfPax=1&System=ABG&lang=en&includeAsyncProducts=true
+```
+{% endcodetab %}
+{% codetab JSON %}
+```
+https://api.holidayextras.co.uk/v1/carpark/MUC.js?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2017-12-01&ArrivalTime=1200&DepartDate=2017-12-08&DepartTime=1200&NumberOfPax=1&System=ABG&lang=en&includeAsyncProducts=true
```

Comment:
Tiny thing - can you update both this request URL and the one above to have dates in 2025? May as well make it a bit more up-to-date at least for the first partners to look at this.

Reply:
adjusted